ports/apriltag/portfile.cmake

```
@@ -6,14 +6,17 @@ vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO AprilRobotics/apriltag
     REF v${VERSION}
-    SHA512 45965f2f888475ea5aecdada0d41a5495f6f0c545d5219c96d82aea009e310a17308df858d4ba40ff191cfe15801dfe16298f20efcc5c117695dd0fa8d3a3fd7
+    SHA512 c6efa98b8c85702441b421955a8fd78cb99ff07e014da5593b95246a0aaf391e9959d336d8005ba193fb6dad0fb5858852918087a401e180af9c46df38a900aa
     HEAD_REF master
     PATCHES
         fix-config.patch
 )
 
 vcpkg_cmake_configure(
     SOURCE_PATH "${SOURCE_PATH}"
+    OPTIONS
+        -DBUILD_PYTHON_WRAPPER=OFF
+        -DBUILD_EXAMPLES=OFF

```

```suggestion
        -DBUILD_EXAMPLES=OFF
        -DBUILD_TESTING=OFF
```
and drop the patch? (If BUILD_TESTING doesn't already default to OFF).

---

There is no such option as `BUILD_TESTING` in the CMakeLists.txt file

---

It doesn't need to be an explicit `option`. CMake users can add any variable as a `cache` variable. (However, normal variables hide cache variable.) apriltag's CI scripts set this variable to ON.